cdap-api/src/main/java/co/cask/cdap/internal/workflow/ProgramWorkflowAction.java

```diff
@@ -74,9 +87,29 @@ public void initialize(WorkflowContext context) throws Exception {
              programType != null ? programType.name() : null, programName);
   }
 
+  public void setTxClient(TransactionSystemClient txClient) {
```

Comment:
Do we need this method, since the txClient is already set in the constructor, right?